<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/SAGis_Logo.ico</normaloff>../resources/SAGis_Logo.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="toolBar" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonFirst">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Erster Datensatz</string>
        </property>
        <property name="text">
         <string>|&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonPrevious">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Vorheriger Datensatz</string>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="labelIndex">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>0/0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonNext">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Nächster Datensatz</string>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonLast">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Letzter Datensatz</string>
        </property>
        <property name="text">
         <string>&gt;|</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonShowCurrent">
        <property name="minimumSize">
         <size>
          <width>86</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Objekt in der Karte zeigen</string>
        </property>
        <property name="text">
         <string>Objekt anzeigen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonShowAll">
        <property name="minimumSize">
         <size>
          <width>106</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Alle Objekte in der Karte zeigen</string>
        </property>
        <property name="text">
         <string>Alle Objekt anzeigen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonExportExcel">
        <property name="toolTip">
         <string>Exportieren (Excel Format)</string>
        </property>
        <property name="text">
         <string>Exportieren (Excel Format)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSagisweb">
        <property name="toolTip">
         <string>In geöffneter SAGis web Karte anzeigen</string>
        </property>
        <property name="text">
         <string>In SAGis web anzeigen</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonClose">
        <property name="text">
         <string>Schließen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
